### PR TITLE
Implemented Design for Orientation Pages

### PR DESCRIPTION
**Pull request by mariiadevphc**

### CODE REVIEW

File: frontend/src/pages/Orientation/Pages/Bio.tsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React, { useState } from "react";
+import '../Orientation.css';
 import './Bio.css';
 
+
 const MAX_CHAR_COUNT = 1000;
 
 function Bio(props: any) {
     const [biography, setBiography] = useState(props.user?.biography);
+    const [location, setLocation] = useState(props.user?.location);
```
Comment (nick-cd) [inline frontend/src/pages/Orientation/Pages/Bio.tsx:10]: Do you plan to use the user's location for something?

**Review by nick-cd — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Orientation/Pages/RelativeLocation.tsx [hunk 2]
```diff
@@ -109,18 +110,25 @@ function RelativeLocation(props: any) {
     }
 
     return (
-        <div>
-            <p>Where is your community?</p>
-            {/* TODO: Find out how to place a space in the middle of the postal code mask without introducing inconsistencies */}
```
[the remark this answers is not in the document]
Comment (christian-phc) [reply frontend/src/pages/Orientation/Pages/RelativeLocation.tsx:114]: Must have overlooked this. Fix can be found on commit [ee9e367](https://github.com/projecthuman-repo/Coquest-WebApp/pull/51/commits/ee9e3672b9e69743a7cc79d1448f7003904891c1). Now it visually adds the space for the user and should show the same results as the ones pre-fix.

File: frontend/src/pages/Orientation/Pages/Bio.tsx [hunk 3]
```diff
@@ -16,17 +19,23 @@ function Bio(props: any) {
...
-
-                <textarea
-                    className="bio-input"
-                    placeholder="Add your bio"
-                    value={biography}
-                    onChange={onEdit}
-                    maxLength={MAX_CHAR_COUNT}>
-                </textarea>
-                <p>{biography.length}&nbsp;/&nbsp;{MAX_CHAR_COUNT}</p>
+                <h3 className="main-heading">Welcome to Coquest&#44; {props.user?.name}!</h3>
```
[the remark this answers is not in the document]
Comment (christian-phc) [reply frontend/src/pages/Orientation/Pages/Bio.tsx:22]: Removed break tags in favor of margin. Changes on [a036b03](https://github.com/projecthuman-repo/Coquest-WebApp/pull/51/commits/a036b039ad50a36c41142b88514a207770d7a684).

File: frontend/src/pages/Orientation/Pages/RelativeLocation.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (nick-cd) [reply frontend/src/pages/Orientation/Pages/RelativeLocation.tsx:114]: My apologies. Yes, you are correct. I glossed over some details when reviewing on my phone 😅

File: frontend/src/pages/Orientation/Pages/Bio.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nick-cd) [reply frontend/src/pages/Orientation/Pages/Bio.tsx:22]: Thanks!

I see some straggling break tags in `Orientation.tsx`. May you remove those as well?

File: frontend/src/components/UploadImage/Cropper.tsx [hunk 4]
```diff
@@ -7,8 +7,8 @@ import { Button } from '@mui/material';
 import { gql } from 'graphql-request';
 import graphQLClient from '../../apiInterface/client';
 
-const CROP_WIDTH = 400; 
-const CROP_HEIGHT = 400;
+const CROP_WIDTH = 175;
```
[the remark this answers is not in the document]
Comment (mariiadevphc) [reply frontend/src/components/UploadImage/Cropper.tsx:10]: Changed for consistency. When the user generates an image its dimensions is 175px by 175px, so just to make it consistent and nice, the image's display dimensions are set to be the same.

File: frontend/src/pages/Orientation/Pages/Bio.tsx (hunk 1 above)
Comment (mariiadevphc) [reply frontend/src/pages/Orientation/Pages/Bio.tsx]: Location was implemented on this page before I saw that there's a separate page for it, forgot to remove, thanks for noticing :) Removed now.

File: frontend/src/pages/Orientation/Orientation.tsx [hunk 5]
```diff
@@ -95,37 +98,54 @@ function Orientation() {
 
         const SelectedPageView = RegistrationPages[index].view;
         return (
-            <Container className="orientation">
+            <div className="orientation">
+
+                <br></br>
+                <br></br>
+
+                {/* TODO: would make sense to add a progress bar to show progress in onboarding completion */}
```
[the remark this answers is not in the document]
Comment (mariiadevphc) [reply frontend/src/pages/Orientation/Orientation.tsx]: Implemented with 849a8b5275ec52e33eb78ecbf932e8cd965249b2

File: frontend/src/pages/Orientation/Pages/Bio.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mariiadevphc) [reply frontend/src/pages/Orientation/Pages/Bio.tsx:21]: Removed with 849a8b5275ec52e33eb78ecbf932e8cd965249b2

File: frontend/src/components/UploadImage/Cropper.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (nick-cd) [reply frontend/src/components/UploadImage/Cropper.tsx:10]: Makes sense, thanks!

**Pull request merged**